### PR TITLE
catch the 'no such window' error for a corner case

### PR DESCRIPTION
**Pull request by link89**
fix https://github.com/webdriverio/webdriverio/issues/6515

### CODE REVIEW

File: packages/webdriverio/src/commands/element/click.ts [hunk 1]
```diff
@@ -129,8 +129,14 @@ export default async function click (
                 button
             }]
         }])
-
-        return this.releaseActions()
+        try {
```
Comment (christian-bromann) [inline packages/webdriverio/src/commands/element/click.ts:132]: Can we add a comment as to why we catch this here and ignore the `no such window` error?

**Review by christian-bromann — CHANGES_REQUESTED** [review]
Can we add a unit test for this?

File: packages/webdriverio/src/commands/element/click.ts (hunk 1 above)
Comment (link89) [reply packages/webdriverio/src/commands/element/click.ts:132]: > Can we add a unit test for this?

Any guidelines for the unit tests? I guess I need to mock the action chains request in `got.ts` but not sure how to write it.

Comment (christian-bromann) [reply packages/webdriverio/src/commands/element/click.ts:132]: Have a look at the [existing unit tests](https://github.com/webdriverio/webdriverio/blob/main/packages/webdriverio/tests/commands/element/click.test.ts). Everything is already mocked and you can just add a test for your use case in a similar fashion. Let me know if anything is unclear.

**Comment by christian-bromann** [comment]
@link89 anything I can do to help?

**Comment by christian-bromann** [comment]
ping @link89

**Comment by christian-bromann** [comment]
Closing due to inactivity.

**Pull request closed without merging**